sourced/ml/mining/spiders/cpp_stdlib.py

```diff
@@ -0,0 +1,38 @@
+import scrapy
+
+
+class CppStdlibSpider(scrapy.Spider):
+    """Spider for scraping the C/C++ standard libraries."""
+
+    name = "cpp-stdlib-spider"
+
+    def start_requests(self):
+        """Start making requests."""
+        url = "https://en.cppreference.com/w/%s/header"
+        for lang in ["c", "cpp"]:
+            request = scrapy.Request(url % lang, callback=self._parse)
+            request.meta["lang"] = lang
+            yield request
+
+    def _parse(self, response):
+        """Parse the response for the index."""
+        lang = response.meta["lang"]
+        if lang == "c":
+            default_since = "since C89/90"
+        else:
+            default_since = "since C++95"
+        for block in response.css("tr.t-dsc > td"):
+            library = block.css("b::text").get()
+            if library is None and lang == "cpp":
+                library = block.css("tt::text").get()
+            if library:
+                since = default_since
+                info = []
+                for i in block.css("span::text").getall():
+                    i = i[1:-1]
+                    if i.startswith("since"):
+                        since = i
+                    else:
+                        info.append(i)
+                info.append(since)
+                yield {"library": library[1:-1], "lang": lang, "info": info}
```

Comment:
If you want to trim quotes, `library.strip('"')` should be more obvious.

Reply:
no, it's to trim `<` and `>`, but yeah will use it instead, and above as well for `(` `)`